### PR TITLE
feat(presets): add tfvarsVersions preset

### PR DESCRIPTION
**Pull request by morremeyer**
## Changes

This adds a `tfvarsVersions` preset as discussed in https://github.com/renovatebot/renovate/pull/21994#discussion_r1188303503.

## Context

This change will make it easier to update versions in `.tfvars` files.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/regex-managers.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/regex-managers.ts:45]:
```suggestion
    description: 'Update `*_version` variables in `.tfvars` files.',
```

Let's end the sentence with a full stop.

I'll let the maintainers review the code. 🙂

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by HonkingGoose — APPROVED** [review]
I'm happy with the description now. I'll leave the code review to the maintainers.

**Review by rarkins — APPROVED** [review]

**Pull request merged**
